webpack.config.js

```diff
@@ -5,11 +5,12 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 module.exports = (argv) => {
     const isProduction = argv.mode === 'production';
     return {
-        entry: "./src/index.jsx",
+        entry: "./src/index.tsx",
         output: {
             filename: "main.js",
             path: path.resolve(__dirname, "build"),
         },
+        devtool: "inline-source-map",

```

Comment:
We should think if it's necessary in prod mode.

Reply:
Removed everything related to source map.